core/src/main/java/org/bitcoinj/core/Sha256Hash.java

```diff
@@ -18,7 +18,7 @@
 package org.bitcoinj.core;
 
 import com.google.common.io.ByteStreams;
-import com.google.common.primitives.*;
```

Comment:
This is an unrelated change.

Reply:
It's somewhat related, since I'm trying to identify places in the code that are using Guava functions that are now provided by the JDK. When I looked at this file, I saw it was using `Ints` (only) but not for the methods I was replacing. Do you want me to remove it and send it separately?